### PR TITLE
finished up MathGame

### PR DESCRIPTION
**Pull request by S-Smans**
Finished the project before I forked it. I just renamed the folder and copied it on the cloned repository.

### CODE REVIEW

File: MathGame.S-Smans/MathGame/Helper.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
+﻿
+namespace MathGame;
+
+internal class Helper
+{
+    private List<Game> _games = new List<Game>();
+
+    // Difficulty level of the game
```
Comment (TheCSharpAcademy) [inline MathGame.S-Smans/MathGame/Helper.cs:8]: I strongly recommend you name your variables so you don't have to explain them with comments. If your variable  was called DifficultyLevel, for example, you code exclude the comment. Code should be self-explanatory and comments should only be added when absolutely necessary.

File: MathGame.S-Smans/MathGame/Helper.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    public int lvl = 1;
+
+    // Question amount that are being asked
```
Comment (TheCSharpAcademy) [inline MathGame.S-Smans/MathGame/Helper.cs:11]: Please name your variable in a way that the comment isn't necessary.

File: MathGame.S-Smans/MathGame/Helper.cs [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    public int ask = 5;
+
+    // Create a class that returns 2 random numbers in an array
```
Comment (TheCSharpAcademy) [inline MathGame.S-Smans/MathGame/Helper.cs:14]: Unnecessary comment (the method's name explains enough)

File: MathGame.S-Smans/MathGame/Helper.cs [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        {
+            ask = input;
+            Console.WriteLine($"Successfuly changed question amount to {ask}");
+            Console.ReadLine();
+        } else
+        {
+            Console.WriteLine($"{input} is invalid. Minimum question amount is 1 and Maximum is 10");
+            Console.ReadLine();
+        }
+
```
Comment (TheCSharpAcademy) [inline MathGame.S-Smans/MathGame/Helper.cs:102]: Remove unnecessary empty lines 102,103. As a rule of thumb, have spaces between methods, between groups of statements (such as line 89), between namespaces and classes,  but not between curly brackets

File: MathGame.S-Smans/MathGame/Helper.cs [hunk 5]
```diff
@@ -0,0 +1,105 @@
+﻿
```
Comment (TheCSharpAcademy) [inline MathGame.S-Smans/MathGame/Helper.cs:1]: remove empty line 1

File: MathGame.S-Smans/MathGame/Game.cs [hunk 6]
```diff
@@ -0,0 +1,28 @@
+﻿
```
Comment (TheCSharpAcademy) [inline MathGame.S-Smans/MathGame/Game.cs:1]: Remove empty line 1

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

File: MathGame.S-Smans/MathGame/Helper.cs (hunk 1 above)
Comment (S-Smans) [reply MathGame.S-Smans/MathGame/Helper.cs]: Renamed the public fields in the Pascal case.

File: MathGame.S-Smans/MathGame/Helper.cs (hunk 4 above)
Comment (S-Smans) [reply MathGame.S-Smans/MathGame/Helper.cs]: Went through my source code and added/removed spacing following specified rules

**Comment by S-Smans** [comment]
Do I need to press the re-request after I implemented my changes?

**Pull request merged**